pkg/configconversion/simple.go

```diff
@@ -14,6 +15,9 @@ func ToHTTPServingInfo(in *legacyconfigv1.HTTPServingInfo) (out configv1.HTTPSer
 	if err != nil {
 		return configv1.HTTPServingInfo{}, err
 	}
+	if len(out.CipherSuites) == 0 {
```

Comment:
This is ultimately dead code, but I am surprised that https://github.com/openshift/library-go/blob/master/pkg/config/configdefaults/config_default.go#L53-L55 doesn't cover it.

Reply:
I think once we switch integration to properly use the new configs then yeah it should be good. But for now I'm leaving it as is. I'll open a JIRA for that: https://jira.coreos.com/browse/MSTR-223